docs/Model_Assumptions.rst

```diff
@@ -603,6 +603,13 @@ The perfect foresight can lead to suspicious dispatch of assets, for example cha
 
 .. _limitations-missing-kpi:
 
+
+Optimization precision
+######################
+
+The MVS makes use of the open energy modelling framework (oemof) by using oemof-solph. It is defined that the energy system is solved with the `cbc-solver` and a `ratioGap=0.03`. This influences the precision of the optimized decision variables, ie. the optimized capacities as well as the dispatch of the assets.
+In some cases it was possible to observe that the precision allowed minimal and also negative dispatch or capacities (scale: <10e-6). In case of storage assets, this can lead to SOC curves with negative values or values far above the viable value 1. As the reason for these inconsistencies is known, the MVS enforces minimal capacities and dispatch of 10e-6, ie. all capacities or flows smaller then 10e-9 are set to zero. This is applied to absolute values, so that irregular (and incorrect) values for decision variables can still be detected.
```

Comment:
```suggestion
In some cases the dispatch or capacities varies around 0 with fluctuation of the order of floating point precision (relative scale well below 10e-6). In case of storage assets, this can lead to SOC curves with negative values or values far above the viable value 1. As the reason for these inconsistencies is known, the MVS enforces minimal capacities and dispatch of 10e-6, ie. all capacities or flows smaller then 10e-9 are set to zero. This is applied to absolute values, so that irregular (and incorrect) values for decision variables can still be detected.
```
This is not really clear: the scale you have here is relative, but you mention later that it applies to absolute values? Fluctuations of 1e-15 sum up to make the SOC "far above" 1 ? In the sentence "MVS enforces minimal capacities and dispatch of 10e-6, ie. all capacities or flows smaller then 10e-9 are set to zero" it is implied that because the unit of capacity is kW then 1 kW * 1e-9 = 1e-6 W, however if one does not have this in mind the sentence is quite confusing :)